Solar array edits

cwang2018 opened the pull request.

scuervo101 reviewed (changes requested):
A few changes should be made:

- Remove the video added:AeroBody/SolarArray/Cielo_Media/Cielo_Soldering2.MOV
-  - We have a limited amount of space on the repo and videos will use up a ton of storage. If you want to add a video, send me a message and I can upload it to the sg youtube channel and have a link embedded on the website
- Convert the docs to pdf: AeroBody/SolarArray/Documents/*.docx
-  - .docx is not properly illustrated on the github site. Github will only show the raw binary data of .docx. pdf format does work properly

Other then that, I'd recommend placing markdown files in folders for organization so you can easily see that the manufacturing page has all of the markdown files in a folder as links

cwang2018 commented:
Thanks for the suggestion! I've changed the file types and deleted the video, but I'll fix the file organization later so you can wait until then to approve the merge if you'd like.

Seth10001 commented on AeroBody/SolarArray/Knowledge_Retention.md:
```diff
@@ -0,0 +1,7 @@
+---
+
+
+For more information on IV curves, click [here.](IV_Curves.pdf) 
```
pdf link yields a 404 error. Move the pdf into the `_static` folder and update the link here.

Seth10001 commented on AeroBody/SolarArray/SOPs.md:
```diff
@@ -1,8 +1,12 @@
 Below are the Standard Operating Procedures (SOPs) for Solar Array manufacturing. Make sure you review them as needed before coming into the workshop! 
 
-[Encapsulation SOP](https://dochub.com/bookworm142013/orO7lgeVLJqqWW1RjMP2p5/encapsulation-sop-pdf?dt=igREcRp1CEz3hhAWkhU3)
+[Encapsulation SOP](Documents/Encapsulation_SOP.pdf)
```
pdf link yields a 404 error. Move the pdf into the `_static` folder and update the link here.

Seth10001 commented on AeroBody/SolarArray/Knowledge_Retention.md:
```diff
@@ -0,0 +1,7 @@
+
+For more information on IV curves, click [here.](IV_Curves.pdf) 
+
+For more information on angle calculations, click [here.](Documents/Angle_Calculations.pdf)
```
pdf link yields a 404 error. Move the pdf into the `_static` folder and update the link here.

Seth10001 commented on AeroBody/SolarArray/SOPs.md:
```diff
@@ -1,8 +1,12 @@
-[Encapsulation SOP](https://dochub.com/bookworm142013/orO7lgeVLJqqWW1RjMP2p5/encapsulation-sop-pdf?dt=igREcRp1CEz3hhAWkhU3)
+[Encapsulation SOP](Documents/Encapsulation_SOP.pdf)
+
+[Soldering SOP](Documents/Solar_Cell_Soldering_SOP.pdf)
```
pdf link yields a 404 error. Move the pdf into the `_static` folder and update the link here.

Seth10001 reviewed (changes requested):
See comments about the pdfs. I recommend uploading the video to youtube and link it similar to how we did for the lectures.

See syntax example here: https://raw.githubusercontent.com/Solar-Gators/docs/master/Lectures/index.rst

See live example: https://solar-gators-docs--47.org.readthedocs.build/en/47/Lectures/index.html
